### ISSUE TITLE
VDC disconnected on testnet for no obvious reason

### ISSUE THREAD

**Issue by gmachtel** [threefoldtech/itenv_testnet#15]
A VDC on testnet just got disconnected for no obvious reason (was created on May 26 so not expired).

3Bot: geertincubaid
VDC: silver5

**Comment by hossnys** [comment]
after doing initial investigation on this vdc found that it's master node went down , didn't find any logs about what caused this from my end , here is the vdc report :
![Screenshot from 2021-06-08 14-46-06](https://user-images.githubusercontent.com/26298009/121188330-4cc48180-c869-11eb-919d-3fd95575f6a4.png)

here is the logs from grafana : 
```

&nbsp; | 2021-06-08 12:26:42.858 \| DEBUG    \| jumpscale.sals.kubernetes.manager:_execute:45 - kubernetes manager: kubectl --kubeconfig /root/.kube/config get deployments -A -o json\n
-- | --
&nbsp; | 2021-06-08 12:26:17.161 \| DEBUG    \| jumpscale.sals.kubernetes.manager:_execute:45 - kubernetes manager: kubectl --kubeconfig /root/.kube/config top nodes  --no-headers=true\n
&nbsp; | \n
&nbsp; | 2021-06-08T12:22:44Z &lt;Greenlet at 0x7f4f50927e10: &lt;bound method KubernetesAutoExtend.job of &lt;k8s_auto_extend.KubernetesAutoExtend object at 0x7f4f4d4d4610&gt;&gt;&gt; failed with Runtime\n
&nbsp; | \n
&nbsp; | Unable to connect to the server: dial tcp 185.206.122.28:6443: connect: no route to host\n
&nbsp; | jumpscale.core.exceptions.exceptions.Runtime: Failed to execute: kubectl --kubeconfig /root/.kube/config top nodes  --no-headers=true, error was W0608 12:22:33.366129   26440 top_node.go:119] Using json format to get metrics. Next release will switch to protocol-buffers, switch early by passing --use-protocol-buffers flag\n
&nbsp; | raise j.exceptions.Runtime(f"Failed to execute: {cmd}, error was {err}")\n
&nbsp; | File "/sandbox/code/github/threefoldtech/js-sdk/jumpscale/sals/kubernetes/manager.py", line 240, in execute_native_cmd\n
&nbsp; | return method(self, *args, **kwargs)\n
&nbsp; | File "/sandbox/code/github/threefoldtech/js-sdk/jumpscale/sals/kubernetes/manager.py", line 25, in wrapper\n
&nbsp; | out = self.manager.execute_native_cmd("kubectl top nodes  --no-headers=true")\n
&nbsp; | File "/sandbox/code/github/threefoldtech/js-sdk/jumpscale/sals/vdc/kubernetes_auto_extend.py", line 73, in update_stats\n
&nbsp; | monitor.update_stats()\n
&nbsp; | File "/sandbox/code/github/threefoldtech/js-sdk/jumpscale/packages/vdc_dashboard/services/k8s_auto_extend.py", line 13, in job\n
&nbsp; | File "src/gevent/greenlet.py", line 906, in gevent._gevent_cgreenlet.Greenlet.run\n
&nbsp; | Traceback (most recent call last):\n
&nbsp; | \n
&nbsp; | 2021-06-08 12:22:44.713 \| ERROR    \| domain:job:32 - failed to fetch all deployed domains due to error Failed to execute: kubectl --kubeconfig /root/.kube/config get deployments -A -o json, error was Unable to connect to the server: dial tcp 185.206.122.28:6443: i/o timeout\n
&nbsp; | 2021-06-08 12:22:33.306 \| DEBUG    \| jumpscale.sals.kubernetes.manager:_execute:45 - kubernetes manager: kubectl --kubeconfig /root/.kube/config top nodes  --no-headers=true\n
&nbsp; | 2021-06-08 12:21:17.121 \| DEBUG    \| jumpscale.sals.kubernetes.manager:_execute:45 - kubernetes manager: kubectl --kubeconfig /root/.kube/config get deployments -A -o json\n


```

**Comment by AhmedHanafy725** [comment]
This VDC was deployed before the fix of this issue #11, so the wallet was drained so early and it couldn't keep the ipv4 units in the pool.

**Comment by gmachtel** [comment]
OK will close then and check behavior of newly deployed nodes
